virttest/libvirt_xml/vm_xml.py

```diff
@@ -613,41 +613,44 @@ def vm_rename(vm, new_name, uuid=None, virsh_instance=base.virsh):
         :param vm: VM class type instance
         :param new_name: new name of vm
         :param uuid: new_vm's uuid, if None libvirt will generate.
-        :return: a new VM instance
+        :return: a new VM instance or raise LibvirtXMLError
         """
+        if vm.name == new_name:
```

Comment:
Thanks for your comment.
But Just alter UUID without changing VM name, which will leads error.
[root@localhost avocado-vt]# virsh edit win7
error: operation failed: domain 'win7' already exists with uuid 8f01e012-015c-4622-993b-e5df6fa1f7b7
Failed. Try again? [y,n,f,?]: